Dockerfile.kong

```diff
@@ -10,9 +10,8 @@ COPY build-kong.sh /build-kong.sh
 
 RUN /build-kong.sh
 
-FROM cdrx/fpm-ubuntu:18.04 as FPM
+FROM kong/fpm:0.0.1 as FPM
```

Comment:
need to build our own fpm docker image because there isn't a arm equivalent for cdrx one